sections/exchange/hooks/useExchange.tsx

```diff
@@ -230,13 +237,41 @@ const useExchange = ({
 	]);
 
 	const basePriceRate = useMemo(
-		() => getExchangeRatesForCurrencies(exchangeRates, baseCurrencyKey, selectedPriceCurrency.name),
-		[exchangeRates, baseCurrencyKey, selectedPriceCurrency.name]
+		() =>
+			txProvider === '1inch'
+				? cmcQuotes != null && baseCurrencyKey != null && selectPriceCurrencyRate != null
+					? cmcQuotes[baseCurrencyKey].price / selectPriceCurrencyRate
+					: 0
+				: getExchangeRatesForCurrencies(exchangeRates, baseCurrencyKey, selectedPriceCurrency.name),
```

Comment:
The idea here is that `basePriceRate` and `quotePriceRate` would take the rate from whatever the price provider is... so if its `1inch` we grab the ones from `cmc`, if its `synthetix` we get it using `exchangeRates`... can definitely be improved and abstracted. I'm thinking, perhaps in a later PR where we introduce full trading from any coin to `sUSD`, this can be addressed.